hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonOutputCommitter.java

```diff
@@ -168,6 +168,9 @@ public void commitJob(JobContext context) throws IOException {
           throw new IOException(e);
         }
       }
+      // After merging index, update newMetaEntry with updated merge index size
+      CarbonLoaderUtil
```

Comment:
can check this only if merge index is enabled

Reply:
okay